program/rust/src/tests/pyth_simulator.rs

```diff
@@ -536,15 +536,17 @@ impl PythSimulator {
     /// TODO : this fixture doesn't set the product metadata
     pub async fn setup_product_fixture(
```

Comment:
nit: comment should say `publishers`

Reply:
I believe this comment is correct. there's stuff like symbol name in the test case assets, but we only make it available for lookup to distinguish symbols human-readably, without filling in the relevant field in the metadata k/v/ store.